Makefile

```diff
@@ -1,48 +1,116 @@
 #! /usr/bin/make -f
+
 SHELL:=/bin/bash
 
+define pod
```

Comment:
This is a method for creating documentation in tools like Makefiles, or  things that don't have other methods.